components/centraldashboard/package.json

```diff
@@ -69,12 +81,14 @@
         "@babel/core": "^7.4.4",
         "@babel/plugin-transform-runtime": "^7.4.4",
         "@babel/preset-env": "^7.4.4",
+        "@babel/register": "^7.4.4",
         "@babel/runtime": "^7.4.4",
         "@polymer/test-fixture": "^4.0.2",
         "@types/express": "^4.16.1",
         "@types/gapi.client.monitoring": "^3.0.1",
         "@types/jasmine": "^3.3.12",
         "babel-loader": "^8.0.5",
+        "chai": "^4.2.0",
```

Comment:
I actually tried that initially, however, the issue I had with Jasmine, is that it depends on `window` which leads me to believe that it's only a front-end testing framework.

Reply:
It can definitely be used as a standalone test framework.

I tried `npx jasmine --require="@babel/register" screen-diff-testing/diff-test.js` and after fixing a few things that aren't present in Jasmine (this.timeout, `beforeAll` instead of `before`), it seems to have run for me.

Reply:
Ah, I was initially using `jasmine-ajax` since I didn't bother reading the docs. Thanks, Jason!